pkgs/applications/editors/mmj2/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, maven, fetchFromGitHub, jre, makeWrapper }:
+
+maven.buildMavenPackage {
+  pname = "mmj2";
+  version = "2.5.2";
+
+  src = fetchFromGitHub {
+    owner = "digama0";
+    repo = "mmj2";
+    fetchSubmodules = true;
+    rev = "1cd95c1fe4435899c8575644fccb412dd77d79e4";
```

Comment:
This should be the version and not a commit hash unless you try to add the package as unstable.

Reply:
Thanks for the review! How would I try to add the package as unstable? The latest release is from 2017 and uses a different build process and doesn't have the wrapper script.

Reply:
Maybe my explanation was unclear but you append `-unstable-yyyy-my-dd` to you version maybe with a comment why you package unstable and not the stable version. (: